### PR TITLE
chore: add merkle dapp

### PR DESCRIPTION
**Pull request by MuckT**
> To make a submission to the Dapps Page, please complete the task lists
> below; otherwise remove them.

Your Dapp:

- [x] connects to Valora via WalletConnect V2;
- [x] must be mobile-first or mobile responsive;
- [x] is in one of the existing categories; and
- [x] includes a link to user documentation.

For more information see the [Valora Listing Overview](https://docs.valora.xyz/listing/overview).

### CODE REVIEW

File: src/valora-dapp-list.json [hunk 1]
```diff
@@ -394,6 +394,15 @@
       "url": "https://core.allbridge.io/",
       "listOnAndroid": true,
       "listOnIos": true
+    },
+    {
+      "name": "Merkle",
+      "id": "merkle-angle",
+      "categoryId": "lend-borrow-earn",
+      "categories": ["lend", "borrow", "earn"],
+      "url": "https://merkl.angle.money/",
```
Comment (jeanregisser) [inline src/valora-dapp-list.json:403]: I saw https://app.merkl.xyz/ is another way to reach the same dapp.
I wonder which one they consider they'd recommend.

**Review by jeanregisser — APPROVED** [review]
👍

File: src/valora-dapp-list.json (hunk 1 above)
Comment (jeanregisser) [reply src/valora-dapp-list.json:403]: I guess according to https://docs.merkl.xyz/, https://app.merkl.xyz/ should be preferred?

Comment (MuckT) [reply src/valora-dapp-list.json]: Fixed in f5a973534f43b668b54f71252d361f4cab278f31!

**Pull request merged**